tools/buildPackage.js

```diff
@@ -147,11 +147,17 @@ if (isLinux) {
 } else if (isDarwin) {
   const macAppName = `${appName}.app`
   cmds.push('ncp ./app/extensions ' + path.join(buildDir, macAppName, 'Contents', 'Resources', 'extensions'))
+  cmds.push('cp ./res/Brave-Browser-0.56.9.pkg ' + path.join(buildDir, macAppName, 'Contents', 'Resources', 'Brave-Browser.pkg'))
```

Comment:
Users update cadence is different, can we download the latest pkg/installer instead of embedding in the installer in resources? Users might end up installing an older version if the users update muon after a new version of brave-core is released.

Reply:
That would be nice- but since the brave-core has auto-update enabled, I don't think it's worth the extra complexity (ex: wget, have user wait, show a loading screen, then attempt).  Currently, install takes place when Muon is launched before any UI is shown. Windows has a stub installer which will fetch the latest, but that is very slow for folks outside the US (there is a known issue for that)